src/routes/SinglePage.js

```diff
@@ -0,0 +1,29 @@
+import { useParams } from 'react-router-dom';
+
+const aboutData = [
+  {
+    slug: 'about-app',
+    title: 'About the app',
+    description:
+      "This application lets us add to-dos, edit, and delete items. Log in to see the delete feature. It also persists to-dos in the browser's local storage for a subsequent visit.",
```

Comment:
[ ] Optional: It is fine for now but when you use liners, it will show an error as it does not allow to use of more than 50 words in a single line. So it would be better to be linter friendly from now  😳